index.html

```diff
@@ -1,12 +1,250 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Esport Events</title>
+    <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tooltipster/2.3.0/css/reset.min.css"> -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <link rel="stylesheet" href="assets/css/index.css">
+    
+    
 </head>
+
 <body>
-    
+    <nav id="navbar-desk" class="nav-desk isDesktop">
+        <div class="dark-bar isDesktop">
+            <div class="bar-content">
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                    <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Login</a>
+        </div>
+        </div>
+        <div class="desktop-nav isDesktop" >
+           <a href="/index.html" class="head-logo-link"> <div class="head-logo">
+                <i class="fa-solid fa-gamepad fa-2xl logo-icon"></i> <h5 class="title">E-Sport <br> Events</h5>
+            </div></a>
+            <ul class="nav-links-desk">
+                <li><a class="orangecolor" href="/pages/about.html">About</a></li>
```

Comment:
- [ ] **Required:** When you click on the About `Nav-link`, the  `About` should be hidden for `Home` to display. Currently, the only way to navigate back to home is through the site logo which isn't good for UX.